### PR TITLE
Add HELM charts

### PR DESCRIPTION
**Pull request by lvarin**
Adding HELM charts. It mostyl only copies the Jinja templates and changes variable references.

### CODE REVIEW

File: charts/tesk/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Deployment instructions for TESK
+
+Helm chart and config to deploy the TESK service. Tested with Helm v3.0.0.
```
Comment (shukapoo) [inline charts/tesk/README.md]: Also tested with "v3.2.0"

Comment (lvarin) [reply charts/tesk/README.md]: I added it, thanks

File: charts/tesk/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Usage
+
+First you must create a namespace in Kubernetes in which to deploy TESK. The
+command below creates all deployments in the context of this namespace. How
+the namespace is created depends on the cluster, so it is not documented here.
+
+To deploy the application:
+ * modify [`values.yaml`](values.yaml)
+ * Create a `secrets.yaml` file. You need to fill up the `username` and `password` of the ftp server configured in WES. If you activated authentication (auth.mode == 'auth' in `values.yaml`), you need as well the `client_id` and `client_secret`, otherwise the auth section must be removed:
```
Comment (shukapoo) [inline charts/tesk/README.md]: If you want to set TESK as an OIDC client to get user tokens, turn on its authentication (auth.mode == 'auth' in `values.yaml`). Please supply OIDC `client_id` and `client_secret` to this. (PLEASE NOTE: If you have another OIDC client for fetching user's OIDC token & don't want to use TESK as client, remove/comment `auth` section)

File: charts/tesk/scripts/configure [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (shukapoo) [inline charts/tesk/scripts/configure]: Is this file really needed? As we don't have any Jinja templates used with the Helm charts

File: charts/tesk/templates/ingress/default-server-secret.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{{ if and (eq .Values.clusterType "kubernetes") .Values.kubernetes.deploy_ingress }}
```
Comment (shukapoo) [inline charts/tesk/templates/ingress/default-server-secret.yaml:1]: Bit confused, what this if condition does

Comment (shukapoo) [reply charts/tesk/templates/ingress/default-server-secret.yaml:1]: Pls add a comment in yaml

File: charts/tesk/README.md (hunk 2 above)
Comment (lvarin) [reply charts/tesk/README.md]: `auth.mode` activated both the OIDC client and the requirement to provide a token to interact to TESK. Or in other workds, if `auth.mode` is set to `noauth`, then TESK has anonymous access.

I will write a mash up of your suggestions and what I just explained.

File: charts/tesk/scripts/configure (hunk 3 above)
Comment (lvarin) [reply charts/tesk/scripts/configure]: Left behind from previous deployment, good catch. I will delete it.

File: charts/tesk/templates/ingress/default-server-secret.yaml (hunk 4 above)
Comment (lvarin) [reply charts/tesk/templates/ingress/default-server-secret.yaml:1]: Not a loop, but a double condition. It is confusing because it is prefix, but in infix is:

if ((.Values.clusterType == 'Kubernetes') and .Values.kubernetes.deploy_ingress == True )

**Review by shukapoo — CHANGES_REQUESTED** [review]
Added few comments which we can address jointly

File: charts/tesk/templates/ingress/default-server-secret.yaml (hunk 4 above)
Comment (lvarin) [reply charts/tesk/templates/ingress/default-server-secret.yaml:1]: Added

Comment (shukapoo) [reply charts/tesk/templates/ingress/default-server-secret.yaml:1]: thnx!

**Review by shukapoo — APPROVED** [review]
All comments resolved & tested these helm charts with OKD "v3.11" & helm "v3.2.0"
Approving
